charts/stable/sabnzbd/values.yaml

```diff
@@ -2,23 +2,90 @@ image:
   repository: ghcr.io/onedr0p/sabnzbd
   pullPolicy: IfNotPresent
   tag: 4.2.2@sha256:5b1f15c8d25afa2342505492486361d4ecf14449e1453aeea45cd7c1e5eeb63a
+exportarrImage:
+  repository: ghcr.io/onedr0p/exportarr
+  pullPolicy: IfNotPresent
+  tag: v1.6.0@sha256:b522e128509b766cf66f847190fccd225ff194dc76775fe6a215f2dd3010cac2
+securityContext:
+  container:
+    readOnlyRootFilesystem: false
+    runAsGroup: 0
+    runAsUser: 0
```

Comment:
We don't want you to run everything as root, unless its rquired.
Considering this ran without root before, this is not needed for the main container.